server/.vscode/launch.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
This looks auto generated and also it is IDE specific, should it really be on github?

Reply:
I think they usually commit these files.
We can delete it, I just removed some unneccesary configs, so it can be generated again.

(PS: I cannot delete it until Wednesday. So go ahead if you have time.)

Reply:
No need then, I just wasn't sure.